### DISCUSSION TITLE
The execution time is increased by approximately 30% from cy2 migration to cypress-cloud

### DISCUSSION THREAD

**Discussion by fabienp38** [currents-dev/cypress-currents-reporter#146] [Q&A] [answered]
Hello, currently My version of cypress is 12.12 and I use the sorry cypress dashboard an I migrate to the last version of cypress-cloud. Since I migrated I notice approximately a 30% increase in test execution time. Is this something known?

**Comment by agoldis** [comment, marked as answer]
https://currents.dev/readme/integration-with-cypress/cypress-cloud#batched-orchestration
